assets/helpers/flashSale.js

```diff
@@ -135,6 +135,64 @@ const Sales: Sale[] = [
 
     },
   },
+  {
+    subscriptionProduct: 'Paper',
+    activeRegions: ['GBPCountries'],
+    startTime: new Date(2019, 0, 5).getTime(), // 4 Jan 2019
```

Comment:
We know the print flash sale should start sometime tomorrow.

However, the upcoming print flash sale will require changes /uk/subscribe/paper (to follow) and those changes can bump the flash sale date to be valid so that the flash sale doesn't appear on /uk and /uk/subscribe but not /uk/subscribe/paper

Reply:
I'm actually talking about the end date, it should be 4th of Feb not 3rd because the time it will finish is 0:00 in the morning